CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 #### Added
 
+- Commercial transfer agreement added to legal docs
```

Comment:
Yesterday we added a separate content section to the changelog to tidy things up a bit

Reply:
Ah no worries, I thought as we were adding this new question it deserved to go in the added section. But you think it should be in content?

Reply:
Um, good question. Its both: 
- an added feature
- a content change

But maybe because it comes under the remit of content stuff, it should go in the content section 🤷